db/schema.rb

```diff
@@ -627,6 +629,8 @@
     t.datetime "updated_at", precision: nil, null: false
     t.datetime "bounced_at", precision: nil
     t.datetime "reminder_sent_at", precision: nil
+    t.datetime "expiration_notice_sent_at", precision: nil
```

Comment:
Whoops, this should not have made it to `main`. I will make a PR

Reply:
Opened #9584

Reply:
(Not blocking release)

Reply:
cherry-picked as 76ad54be5e736d7efbba140748197e0830247712